app/components/person-search-by-name.js

```diff
@@ -26,20 +26,28 @@ export default class PersonSearchByNameComponent extends Component {
       dataMapping: (data) => {
         const entry = data.attributes;
         return {
-          given_name: entry.given_name,
-          family_name: entry.family_name,
+          given_name: entry.given_name.trim().toLowerCase(),
+          family_name: entry.family_name.trim().toLowerCase(),
         };
       },
     });
-
-    result = new Set(result.toArray());
+    result = result
+      .toArray()
+      .filter(
+        (v, i, a) =>
+          a.findIndex((v2) =>
+            ['family_name', 'given_name'].every((k) => v2[k] === v[k])
+          ) === i
+      );
     if (searchParams.trim() !== '' && result) {
       let param_object = {
         family_name: '',
         given_name: '',
       };
       param_object[fieldName] = searchParams.trim();
-      return [...[param_object], ...result];
+      const arr = [...[param_object], ...result];
+      console.log(arr);
```

Comment:
This one should probably be removed ? :p

Reply:
oops 😱